tmt/base.py

```diff
@@ -2006,15 +2010,15 @@ def create(
                 'link': links.to_spec()
                 })
 
-        for name in names:
-            (directory, plan) = os.path.split(name)
-            directory_path = path / directory.lstrip('/')
-            has_fmf_ext = os.path.splitext(plan)[1] == '.fmf'
-            plan_path = directory_path / (plan + ('' if has_fmf_ext else '.fmf'))
+        for plan_name in names:
+            plan_path = path / Path(plan_name).unrooted()
+
+            if plan_path.suffix != '.fmf':
+                plan_path = plan_path.parent / f'{plan_path.name}.fmf'
```

Comment:
could use `with_suffix('.fmf') instead of f-string. Not sure if it has any benefit, just saying :)

Reply:
`plan_path = (plan_path.parent / plan_path.name).with_suffix('.fmf')` certainly looks less readable to me :/

Reply:
you're right.